### PR TITLE
docs: update readme for rebranding

### PR DESCRIPTION
**Pull request by AdnoC**
## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [x] Code is reviewed for security

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,30 +30,15 @@ Include this library as a test-scoped dependency in your POM. Ensure the `versio
...
-
-- `options` wires a JSON string to aXe, allowing rules to be toggled on or off. See the `testAccessibilityWithOptions` unit test for a sample single-rule execution, and the [axe-core API documentation](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#b-options-parameter) for full documentation on the options object. The runOnly option with tags may be of particular interest, allowing aXe to execute all rules with the specified tag(s).
-- `include` adds to the list of included selectors. If you do not call `include` at all, aXe will run against the entire document.
-- `exclude` adds to the list of excluded selectors. Exclusions allow you to focus scope exactly where you need it, ignoring child elements you don't want to test.
-- `skipFrames` prevents aXe to be recursively injected into all iframes.
-- `analyze` executes aXe with any configuration you have previously defined. If you want to test one or more `WebElement`s, you may pass them into `analyze` instead of using `include` and `exclude`.
```
Comment (WilcoFiers) [inline README.md:55]: Why did you take this out?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# aXe Selenium (Java) Integration
+# axe-core Selenium (Java) Integration
```
Comment (jeankaplansky) [inline README.md:1]:
```suggestion
# axe DevTools Selenium (Java) Integration
```

Comment (jeankaplansky) [reply README.md:1]: Pretty sure it's supposed to be axe DevTools with the axe all lowercase.

File: README.md (hunk 1 above)
Comment (jeankaplansky) [reply README.md]: Ditto. Also, you have instances of axe all over this example that need be all lowercase.

File: README.md [hunk 3]
```diff
@@ -31,30 +30,15 @@ Include this library as a test-scoped dependency in your POM. Ensure the `versio
...
-- `include` adds to the list of included selectors. If you do not call `include` at all, aXe will run against the entire document.
-- `exclude` adds to the list of excluded selectors. Exclusions allow you to focus scope exactly where you need it, ignoring child elements you don't want to test.
-- `skipFrames` prevents aXe to be recursively injected into all iframes.
-- `analyze` executes aXe with any configuration you have previously defined. If you want to test one or more `WebElement`s, you may pass them into `analyze` instead of using `include` and `exclude`.
-
-The aXe documentation should be consulted for more details on customizing and analyzing calls to `axe.run`.
+The `AxeBuilder` type is the main interface. Pass it a Selenium `WebDriver` instance, configure it,
```
Comment (jeankaplansky) [inline README.md:40]: is the `AxeBuilder` cased this way in code? If not, or if it doesn't matter then this should be `axeBuilder`

Comment (AdnoC) [reply README.md:40]: It is. The class is named `AxeBuilder`.

Comment (jeankaplansky) [reply README.md:40]: 👍

File: README.md (hunk 1 above)
Comment (jeankaplansky) [reply README.md]: Adam says this was all replaced with: https://github.com/dequelabs/axe-core-maven-html/pull/87/commits/1b2d4e097d91b5e0cd3be49853b2e63cbf3798a4

Comment (AdnoC) [reply README.md]: The section is no longer true (due to the api changes). Replaced it.

**Pull request merged**